requirements.txt

```diff
@@ -4,11 +4,12 @@ gunicorn==20.1.0
 pandas==1.3.2
 pdf2image==1.16.0
 pydantic==1.8.2
-pytesseract==0.3.8
+pytesseract==0.3.7
```

Comment:
Are we intentionally moving from 0.3.8 to 0.3.7?

Reply:
Ah, forgot I did that. I'm not sure why, but I can't get pytesseract to run in Docker with version 0.3.8, but 0.3.7 works just fine. This can be changed back at any point, sorry for the mix up.

Reply:
0.3.7 is the one that works with the Docker setup we have